src/index.html

```diff
@@ -7,7 +7,9 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1 class="logo">MA</h1>
+    <a href="#logo" >
+      <h1 class="logo" id="logo">MA</h1>
+    </a>
```

Comment:
You don't need t change the html structure

Reply:
Not fixed. There is no need to change HTML in this task